client/my-sites/importer/site-importer/index.jsx

```diff
@@ -100,23 +99,20 @@ class SiteImporter extends React.PureComponent {
 
 		return (
 			<Card className={ cardClasses } { ...( showStart ? cardProps : undefined ) }>
-				<ImporterHeader
-					importerStatus={ state }
-					{ ...{ icon, title, description, isEnabled, site } }
-				/>
-				{ includes( importingStates, state.importerState ) && (
+				<ImporterHeader importerStatus={ importerStatus } { ...{ icon, title, description } } />
```

Comment:
Nit: I think passing these specifically as separate props would be a bit more readable than creating an object and then spreading it.

Reply:
Both `ImporterHeader` instances got fixed in e97f017640bc1311551b6e1f7f0fd2b4a239c986